Policies/KeyVault/Azure_KeyVault_NetSec_Disable_Public_Network_Access/Readme.md

```diff
@@ -0,0 +1,23 @@
+## Azure_KeyVault_NetSec_Disable_Public_Network_Access
+
+### DisplayName 
+[Key Vault must have public access disabled](../../../Control%20coverage/Feature/KeyVault.md#azure_keyvault_netsec_disable_public_network_access)
+
+### Required Policies
+Control can be covered with the below mentioned Azure policy.
+
+#### Policy Details
+Following policy can be used with 'Audit' effect to view the compliance of Key Vaults having `Public Network Access` disabled at the policy assignment scope.
```

Comment:
Following policy can be used with 'Audit' effect to detect non-compliance resources.

Reply:
Addressed.